code/renderers/svelte/src/portable-stories.ts

```diff
@@ -10,6 +10,7 @@ import type {
   Store_CSFExports,
   StoriesWithPartialProps,
   ComposedStoryFn,
+  NamedOrDefaultProjectAnnotations,
```

Comment:
Why did we change to this type?

Reply:
@yannbf ?

Reply:
It's a fix that was done in React but not in other renderers, by mistake. To allow users to use preview.ts with both named export or default export formats:

```ts
export const decorators = []
```

```ts
const preview = {
  decorators: []
]
export default preview
```